nationtweaker.js

```diff
@@ -61,6 +66,27 @@ var tweaks = [
 		"suggestLink": "http://nationbuilder.com/ineffyble/include_in_top_nav_should_be_unticked_by_default_when_creating_new_pages",
 		"function": "stopNewPagesAddedToNavByDefault",
 		"matches": [paths.pageNew]
+	},
+	{
+		"name": "Increase size of email preview windows",
+		"description": "Make email previews full width and 600px high",
+		"suggestLink": "#",
```

Comment:
Hmm, I might need to make proper handling for cases where the tweak doesn't have a suggestion on NationBuilder's suggestion forum (e.g. don't show a "Tell NationBuilder" link on settings). That said, do any of these have a suggestion ticket?